notes/returned-mail.vbs

```diff
@@ -214,20 +213,43 @@ IF ADDR_actions <> "no response received" THEN
     		IF mail_line_one <> "" THEN
     			IF mailing_address_confirmed = "Select:" THEN err_msg = err_msg & vbCr & "Please confirm if the mailing address is the address that the agency attempted to deliver mail to."
     		END IF
-    		IF mailing_address_confirmed = "NO" and residential_address_confirmed = "NO" and notes_on_address = "" THEN  err_msg = err_msg & vbCr & "Please confirm what the address was using notes on address that the agency attempted to deliver mail to. ADDR will not be updated at this time. Please explain where the address was found and on what date."
-    		IF returned_mail = "" THEN  err_msg = err_msg & vbCr & "Please explain what mail was returned."
-    		IF isdate(due_date) = FALSE THEN err_msg = err_msg & vbnewline & "Please enter the verifications requested due date."
+			IF residential_address_confirmed = "YES" and mailing_address_confirmed = "YES" THEN err_msg = err_msg & vbCr & "Please confirm what the address the agency attempted to deliver mail to."
+    		IF mailing_address_confirmed = "NO" and residential_address_confirmed = "NO" THEN err_msg = err_msg & vbCr & "Please confirm what the address the agency attempted to deliver mail to."
+    		IF returned_mail = "" THEN err_msg = err_msg & vbCr & "Please explain what mail was returned."
```

Comment:
Line 218 and line 220: Best practice is to add a trim to the variable.

Reply:
added trim trim(returned_mail) and trim(verifications_requested) the length may only be 3 characters I will discuss further upon review

Reply:
also added a length requirement - often people will only put "SVF"